jukebox/src/main/webapp/js/script.js

```diff
@@ -1,25 +1,112 @@
-function generateGroup() {
+/**
+This file contains all database reads/writes as well some minor webpage
+modifications. It's most likely not secure to have the database available
+on the client-side, so this set-up is temporary (although will likely remain
+as apart of the MVP)
+ */
 
-    //send to server once database schema decided
-    var groupName = document.getElementById('group_name').value;
-    window.location.href = '/create_pin.html';
+//Setting up Firebase configuration
+var firebaseConfig = {
+
+    apiKey: "AIzaSyDkEJE1taOS3R_dxakJjMB98I-3Apjuhxk",
+    authDomain: "jukebox-sps.firebaseapp.com",
+    databaseURL: "https://jukebox-sps.firebaseio.com",
+    projectId: "jukebox-sps",
+    storageBucket: "jukebox-sps.appspot.com",
+    messagingSenderId: "729977109157",
+    appId: "1:729977109157:web:0f47881639cc145866ab4c",
+    measurementId: "G-9M791HTW5T"
+}
+
+// Initialize Firebase
+firebase.initializeApp(firebaseConfig);
+var db = firebase.firestore();
+
+//This function sets up the room page, getting the room name based on PIN
+function presentRoom(isHost) {
```

Comment:
hmm...I think we'd need to look into ensuring that we're loading whether they're the host or not from Firebase rather than just based on what page they hit

Reply:
Yeah, I think that'd be a better idea. I'd need to look into an anonymous user authentication system, so I can store the users of each room and associate and keep track of the host. I think Firebase has some anonymous auth stuff, so I'll try and look into that